### ISSUE TITLE
[Infrastructure] Allow non-host players to see (but not edit) game customization

### ISSUE THREAD

**Issue by ProspectPyxis** [freeboardgames/FreeBoardGames.org#970]
**Is your feature request related to a problem? Please describe.**
I've been working with the game customization interface for a bit now, and I've noticed that there's currently no way for non-host players to even see what customizations have been put in for the game. This could be a problem for public games where there's no one agreed-upon setting for everything.

**Describe the solution you'd like**
My suggestion is to add a fourth property to `GameCustomizationProps` - something like `isHost` - that the developer can then use to disable or enable various elements as necessary. The ideal use case for this is to enable all the interactive elements if the player is a host, but to disable them and set them to read-only if the player is not. Then, allow non-host players to open the configuration pop-up as well.

**Describe alternatives you've considered**
Another possible option is to have a separate and fully static configuration page show up for non-host players instead, though to me, this could lead to violating DRY if the developer would just want to display the same things twice, just with the elements disabled/enabled accordingly.

**Comment by vdfdev** [comment]
Good idea and good point. I don't think it should be too hard to do. I will take a stab at it :)
